contracts/sol6/KyberNetwork.sol

```diff
@@ -1019,7 +942,7 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
 
         for (uint256 i = 0; i < reservesData.isEntitledRebateFlags.length; i++) {
             if (reservesData.isEntitledRebateFlags[i]) {
-                rebateWallets[_index] = reserveRebateWallet[address(reservesData.addresses[i])];
+                rebateWallets[_index] = kyberStorage.getRebateWallet(address(reservesData.addresses[i]));
```

Comment:
lets do this in one contract call.

prepare a list of reserves
and query wallet list after reserve list is ready.
so we can save a few extra contract calls.

Reply:
Change to get list of rebateReserveIds, then call storage once to get list of rebate wallets